src/features/unified-share-modal/EmailForm.js

```diff
@@ -338,7 +338,7 @@ class EmailForm extends React.Component<Props, State> {
                             <FormattedMessage {...commonMessages.cancel} />
                         </Button>
                         <PrimaryButton
-                            isDisabled={submitting || selectedContacts.length === 0 || contactsFieldError} // Check selectedContacts.length === 0 for initial render when contactsFieldError is empty
```

Comment:
The new directive is to keep form submit buttons enabled, even if form is not yet valid. This makes it easier to discover where the errors in the form are (because once you submit an invalid form you then see the errors inline on invalid fields), and this also improves keyboard a11y because you can focus on the button and activate it (and then get read the invalid field in some cases).